subtrack.Tests/SubscriptionCalculatorTests/GetSubscriptionListByMonthTests.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using subtrack.DAL.Entities;
+using subtrack.MAUI.Services;
+using System.Collections;
+
+namespace subtrack.Tests.SubscriptionCalculatorTests
+{
+    public class GetSubscriptionListByMonthTests
+    {
+        [Fact]
+        public void GetSubscriptionListByMonth_MonthProvided_ReturnsUnpaidSubscriptions()
```

Comment:
Make this two test cases instead and create the list of subscriptions directly here.

_MonthProvided_ShouldNotReturn_PaidSubscriptions